src/graphql/schema/Trip/typeDefs.gql

```diff
@@ -96,7 +96,7 @@ input DestinationInput {
 }
 
 input BudgetObjectInput {
-  suggestions: [BudgetInput!]!
+  suggestions: [BudgetInput]!
```

Comment:
Yeah, good change. We could also change it on 
``` type Destination {
  id: ID!
  name: String!
  voters: [User!]!
  creator: User!
}``` because you can change your opinion even if you suggested a destination, so a destination has 0 votes

same for Budget and Timeframe

Reply:
can you change it this PR and push it? Then I can merge it tomorrow